index.js

```diff
@@ -90,6 +91,16 @@ function iconfontCSS(config) {
 			codePoint: currentCodePoint
 		});
 
+		if (config.aliases[fileName]) {
+			_.each(config.aliases[fileName], function(_alias) {
+				glyphMap.push({
+					fileName: _alias,
+					codePoint: currentCodePoint,
+					aliasRoot: fileName // used for less and scss
```

Comment:
Tiny detail: Took me some time to figure out the reason for this, would you mind renaming it to `originalFileName` or something?